lib/main.coffee

```diff
@@ -195,9 +202,10 @@ module.exports =
         pep8warn = atom.config.get('linter-flake8.pep8ErrorsToWarnings')
         flakeerr = atom.config.get('linter-flake8.flakeErrors')
         projDir = @getProjDir(filePath) or path.dirname(filePath)
-        execPath = fs.normalize(@applySubstitutions(atom.config.get('linter-flake8.executablePath'), projDir))
+        execPath = fs.normalize(@applySubstitutions(atom.config.get('linter-flake8.executablePath'),
+          projDir, path.sep))
```

Comment:
Why are you passing `path.sep` in here? Why not just use it directly in `getProjName` as that is the only place it is used?